MediaBrowser.Providers/MediaInfo/AudioImageProvider.cs

```diff
@@ -108,21 +107,19 @@ private async Task<DynamicImageResponse> GetImage(Audio item, List<MediaStream>
             };
         }
 
-        private string GetAudioImagePath(Audio item)
+        private string GetAudioImagePath(Audio item, string newImagePath)
```

Comment:
Get the incoming image that was scanned, Look at the file length+Album and use that for the unique identifier